images/macos/software-report/SoftwareReport.Generator.ps1

```diff
@@ -6,6 +6,7 @@ param (
 
 $ErrorActionPreference = "Stop"
 
+. ("$PSScriptRoot/SoftwareReport.Base.ps1")

```

Comment:
I think we don't need to import `MarkdownPS` anymore, right?

Reply:
yes, removed